build.gradle

```diff
@@ -52,6 +52,7 @@ allprojects {
         threetenabpVersion        =    '1.1.0'
         universalPkgDir           =    "universal"
         composeVersion            =    '1.2.0-alpha08'
+        composeActivityVersion    =    '1.7.2'
```

Comment:
This variable is not being used yet, replace the hardcoded version

Reply:
Checked and modified the version. It is now being used in FluentUI.Demo/build.gradle (line 96). Please check and resolve.